linux/kernel/configuring.md

```diff
@@ -22,12 +22,18 @@ Once you've got everything set up and ready to go, you can compile and run the `
 $ make menuconfig
 ```
 
-If you're cross-compiling,:
+If you're cross-compiling a 32-Bit kernel:
```

Comment:
Lower-case 'b' on 'bit' here (and also below) please :slightly_smiling_face:

Reply:
Fixed in 4ee1b6a